mysql51/src/main/java/another/PropertyHandler.java

```diff
@@ -0,0 +1,10 @@
+package another;
+
+public class PropertyHandler extends mysql51.PropertyHandler {
+
+	PropertyHandler() {
+		super();
+		// TODO Auto-generated constructor stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)